ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/linear_optimisation/LinearProblem.java

```diff
@@ -100,6 +101,30 @@ public MPVariable getRangeActionSetPointVariable(RangeAction rangeAction) {
         return solver.lookupVariableOrNull(rangeActionSetPointVariableId(rangeAction));
     }
 
+    private String rangeActionGroupSetPointVariableId(String rangeActionGroupId) {
+        return rangeActionGroupId + SEPARATOR + VIRTUAL_SET_POINT + SEPARATOR + VARIABLE_SUFFIX;
+    }
+
+    public MPVariable addRangeActionGroupSetPointVariable(double lb, double ub, String rangeActionGroupId) {
+        return solver.makeNumVar(lb, ub, rangeActionGroupSetPointVariableId(rangeActionGroupId));
+    }
+
+    public MPVariable getRangeActionGroupSetPointVariable(String rangeActionGroupId) {
+        return solver.lookupVariableOrNull(rangeActionGroupSetPointVariableId(rangeActionGroupId));
+    }
+
+    public String absoluteRangeActionGroupSetPointConstraintId(RangeAction rangeAction) {
+        return rangeAction.getId() + SEPARATOR + rangeAction.getGroupId().orElseThrow() + SEPARATOR + VIRTUAL_SET_POINT + SEPARATOR + CONSTRAINT_SUFFIX;
+    }
+
+    public MPConstraint addRangeActionGroupSetPointConstraint(double lb, double ub, RangeAction rangeAction) {
+        return solver.makeConstraint(lb, ub, absoluteRangeActionGroupSetPointConstraintId(rangeAction));
+    }
+
+    public MPConstraint getRangeActionGroupSetPointConstraint(RangeAction rangeAction) {
+        return solver.lookupConstraintOrNull(absoluteRangeActionGroupSetPointConstraintId(rangeAction));
+    }
+
```

Comment:
I think the `absolute` in the name of the first method can be removed.

Reply:
done